python/psim/simulation.py

```diff
@@ -53,6 +53,15 @@ def __setitem__(self, name, value):
         """
         self._sim[name] = value
 
+    def get(self, name):
+        """Attempts to retrieve a state field from the underlying simulation.
+        Upon failure, 'None' is returned.
+        """
+        try:
+            return self[name]
+        except RuntimeError:
+            return None
+
```

Comment:
Added a way to get a field without potentially raising an exception.

Reply:
Is this for the PTest integration? Isn't it dangerous for stuff like this to fail silently?

Reply:
This is used, right now, for the snapshot feature that dumps the current state of the simulation upon termination and not in PTest as far as I'm aware.